interfaces/apparmor/backend.go

```diff
@@ -679,6 +679,14 @@ func addContent(securityTag string, snapInfo *snap.Info, cmdName string, opts in
 	}
 	policy = templatePattern.ReplaceAllStringFunc(policy, func(placeholder string) string {
 		switch placeholder {
+		case "###INCLUDEIFEXISTSSNAPTUNING###":
+			features, _ := parserFeatures()
+			for _, f := range features {
+				if f == "include-if-exists" {
+					return "#include if exists \"/var/lib/snapd/apparmor/snap-tuning\""
```

Comment:
Yep this would be a single file - I used this name as that is what @pedronis suggested earlier but I am more than happy to change it if we can come up with something more appropriate.

Reply:
I'm happy for suggestions?  "global-snap-tuning" ?  "all-snap-tuning"?

Reply:
I like global-snap-tuning, but also as I said the best solution to the ambiguity is documentation so this is probably fine as-is too.